### DISCUSSION TITLE
Fuel Gauge with ModelGauge and Low-Battery Alert

### DISCUSSION THREAD

**Discussion by hhl-16** [stm32duino/Arduino_Core_STM32#1882] [Q&A] [answered]
hi, guys
i use max17043 with schematic this
<img width="477" alt="image" src="https://user-images.githubusercontent.com/84115903/204476950-a33057af-d554-4894-aac3-59113f9fb99d.png">
when i use sketch i2c_scanner with arduino -> result is 0x32 (SDA <-> A4, SCL <-> A5)
but when i use stm32 blue pill -> no i2c devices found (SDA <-> PB7, SCL <-> PB6)
thanks

**Comment by fpistm** [comment, marked as answer]
Hi, you have to add Pull up resistor on each I2C lines.
